website_blog_excerpt_img/templates/assets.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Jairo Llopis
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+
+<data>
```

Comment:
@cristinamartinrod ` <data>` --> `<odoo>`

Reply:
`<data>` is also valid and it's preferred by our colleague @Yajo who has been the last to touch this module, due to its neutrality about branding.